mocha.js

```diff
@@ -1999,10 +1999,10 @@ var isatty = tty.isatty(1) && tty.isatty(2);
 exports = module.exports = Base;
 
 /**
- * Enable coloring by default.
+ * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = supportsColor || (process.env.MOCHA_COLORS !== undefined);
+exports.useColors = process.env ? (supportsColor || (process.env.MOCHA_COLORS !== undefined)) : false;
```

Comment:
**style issue:** this line too long(102 chars), imo you should stay with same code-style
e.g: [1](https://github.com/mochajs/mocha/blob/master/lib/reporters/base.js#L111-L114), [2](https://github.com/mochajs/mocha/blob/master/lib/utils.js#L442-L444)